Develop (3da1c6b) fails to build udunits when Python is activated

**Describe the bug**

The Python version is not passed to `update_modules` from `libs/build_udunits.sh` - should it?
```
make[2]: Leaving directory '/glade/scratch/heinzell/hpc-stack/source/hpc-stack-develop-20210511-intel/pkg/udunits-2.2.28/build'
make[1]: Leaving directory '/glade/scratch/heinzell/hpc-stack/source/hpc-stack-develop-20210511-intel/pkg/udunits-2.2.28/build'
+ true
+ update_modules compiler udunits 2.2.28
+ local prefix=/glade/scratch/heinzell/hpc-stack/source/install
+ local modpath=compiler
+ local name=udunits
+ local version=2.2.28
/glade/scratch/heinzell/hpc-stack/source/hpc-stack-develop-20210511-intel/libs/build_udunits.sh: line 4: $4: unbound variable
BUILD FAIL!  Lib: udunits Error:1
```

**To Reproduce**
Add the following to to `config/config_cheyenne_intel.sh`:
```
export HPC_PYTHON="python/3.8.3"

# Load Python environment
module use /glade/p/ral/jntp/tools/miniconda3/4.8.3/modulefiles
module load miniconda3/4.8.3
conda activate ufs-weather-model
```
then run
```
./setup_modules.sh -p /glade/scratch/heinzell/hpc-stack/source/install -c config/config_cheyenne_intel.sh 2>&1 | tee setup_modules.log
./build_stack.sh -p /glade/scratch/heinzell/hpc-stack/source/install -c config/config_cheyenne_intel.sh -y config/stack_noaa.yaml -m 2>&1 | tee build_stack.try1.log
```

**Expected behavior**
Build hpc-stack without errors.

**System:**
Cheyenne/Intel

I got all the way to PIO (2.5.3), this now fails with something I haven't seen before. Have you?
```
make[3]: Entering directory '/glade/scratch/heinzell/hpc-stack/source/hpc-stack-develop-20210511-intel/pkg/pio-2.5.3/build/tests/unit'
/bin/sh ../../libtool  --tag=FC   --mode=compile mpif90 -DHAVE_CONFIG_H -I. -I../../../tests/unit -I../..  -I../../../src/flib -I/glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/include  -fPIC -c -o global_vars.lo ../../../tests/unit/global_vars.F90
libtool: compile:  mpif90 -DHAVE_CONFIG_H -I. -I../../../tests/unit -I../.. -I../../../src/flib -I/glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/include -fPIC -c ../../../tests/unit/global_vars.F90  -fPIC -o .libs/global_vars.o
../../../tests/unit/global_vars.F90(9): error #7013: This module file was not generated by any release of this compiler.   [PIO]
  use pio
------^
../../../tests/unit/global_vars.F90(15): error #6592: This symbol must be a defined parameter, an enumerator, or an argument of an inquiry function that evaluates to a compile-time constant.   [PIO_MAX_NAME]
  integer, parameter :: str_len = pio_max_name, ntest=4
----------------------------------^
../../../tests/unit/global_vars.F90(27): error #6457: This derived type name has not been declared.   [IOSYSTEM_DESC_T]
  type(iosystem_desc_t), save :: pio_iosystem
-------^
../../../tests/unit/global_vars.F90(28): error #6457: This derived type name has not been declared.   [FILE_DESC_T]
  type(file_desc_t), save     :: pio_file
-------^
../../../tests/unit/global_vars.F90(36): error #6592: This symbol must be a defined parameter, an enumerator, or an argument of an inquiry function that evaluates to a compile-time constant.   [PIO_IOTYPE_NETCDF]
  integer, dimension(ntest) :: iotypes =  (/ PIO_iotype_netcdf,         &
---------------------------------------------^
../../../tests/unit/global_vars.F90(36): error #6973: This is not a valid initialization expression.
  integer, dimension(ntest) :: iotypes =  (/ PIO_iotype_netcdf,         &
---------------------------------------------^
../../../tests/unit/global_vars.F90(15): error #6404: This name does not have a type, and must have an explicit type.   [PIO_MAX_NAME]
  integer, parameter :: str_len = pio_max_name, ntest=4
----------------------------------^
../../../tests/unit/global_vars.F90(36): error #6404: This name does not have a type, and must have an explicit type.   [PIO_IOTYPE_NETCDF]
  integer, dimension(ntest) :: iotypes =  (/ PIO_iotype_netcdf,         &
---------------------------------------------^
../../../tests/unit/global_vars.F90(51): error #6404: This name does not have a type, and must have an explicit type.   [PIO_IOTYPE_NETCDF4P]
           (iotype.eq.PIO_iotype_netcdf4p) .or. &
----------------------^
../../../tests/unit/global_vars.F90(52): error #6404: This name does not have a type, and must have an explicit type.   [PIO_IOTYPE_NETCDF4C]
           (iotype.eq.PIO_iotype_netcdf4c) .or. &
----------------------^
../../../tests/unit/global_vars.F90(53): error #6404: This name does not have a type, and must have an explicit type.   [PIO_IOTYPE_PNETCDF]
           (iotype.eq.PIO_iotype_pnetcdf)
----------------------^
compilation aborted for ../../../tests/unit/global_vars.F90 (code 1)
Makefile:492: recipe for target 'global_vars.lo' failed
make[3]: *** [global_vars.lo] Error 1
make[3]: Leaving directory '/glade/scratch/heinzell/hpc-stack/source/hpc-stack-develop-20210511-intel/pkg/pio-2.5.3/build/tests/unit'
Makefile:403: recipe for target 'all-recursive' failed
make[2]: *** [all-recursive] Error 1
make[2]: Leaving directory '/glade/scratch/heinzell/hpc-stack/source/hpc-stack-develop-20210511-intel/pkg/pio-2.5.3/build/tests'
Makefile:449: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/glade/scratch/heinzell/hpc-stack/source/hpc-stack-develop-20210511-intel/pkg/pio-2.5.3/build'
Makefile:379: recipe for target 'all' failed
make: *** [all] Error 2
BUILD FAIL!  Lib: pio Error:2
```

Next error, madis. Seems like the linker order for netcdf and netcdff is reverse of what it should be.
```
a - m_ztopsa.o
mpif90   -fPIC -c madis_ingest.f
mpif90  -L/glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib -L/glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/hdf5/1.10.6/lib   -L/glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/zlib/1.2.11/lib   -fPIC -o ../bin/madis_ingest.exe madis_ingest.o ../lib/madislib.a  -lnetcdf -lnetcdff -lhdf5_hl -lhdf5  -lz -ldl -lm
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_fortv2.o): in function `ncvinq_':
nf_fortv2.f90:(.text+0xd9e): undefined reference to `nc_inq_varndims'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_fortv2.o): in function `ncvpt1_':
nf_fortv2.f90:(.text+0x1222): undefined reference to `nc_inq_varndims'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_fortv2.o): in function `ncvp1c_':
nf_fortv2.f90:(.text+0x16d2): undefined reference to `nc_inq_varndims'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_fortv2.o): in function `ncvpt_':
nf_fortv2.f90:(.text+0x1b99): undefined reference to `nc_inq_varndims'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_fortv2.o): in function `ncvptc_':
nf_fortv2.f90:(.text+0x2483): undefined reference to `nc_inq_varndims'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_fortv2.o):nf_fortv2.f90:(.text+0x2dd0): more undefined references to `nc_inq_varndims' follow
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(module_netcdf_fortv2_c_interfaces.o): in function `netcdf_fortv2_c_interfaces_mp_convert_v2_imap_':
module_netcdf_fortv2_c_interfaces.f90:(.text+0x6d): undefined reference to `nc_inq_vartype'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: module_netcdf_fortv2_c_interfaces.f90:(.text+0x80): undefined reference to `nc_inq_varndims'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: module_netcdf_fortv2_c_interfaces.f90:(.text+0x1b7): undefined reference to `nc_inq_vardimid'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: module_netcdf_fortv2_c_interfaces.f90:(.text+0x21b): undefined reference to `nc_inq_dimlen'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `f2c_v2imap':
nf_v2compat.c:(.text+0x7a): undefined reference to `nc_inq_vartype'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x95): undefined reference to `nc_inq_varndims'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x137): undefined reference to `nc_inq_vardimid'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x166): undefined reference to `nc_inq_dimlen'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncpopt':
nf_v2compat.c:(.text+0x193): undefined reference to `ncopts'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncgopt':
nf_v2compat.c:(.text+0x1a3): undefined reference to `ncopts'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_nccre':
nf_v2compat.c:(.text+0x1c5): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x1e6): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x1f2): undefined reference to `nccreate'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x211): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncopn':
nf_v2compat.c:(.text+0x249): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x25e): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x26a): undefined reference to `ncopen'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x29d): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x2ac): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncddef':
nf_v2compat.c:(.text+0x2c9): undefined reference to `ncdimdef'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x2e1): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncdid':
nf_v2compat.c:(.text+0x2f6): undefined reference to `ncdimid'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x30e): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncvdef':
nf_v2compat.c:(.text+0x32e): undefined reference to `nc_def_var'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x360): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x367): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncvid':
nf_v2compat.c:(.text+0x396): undefined reference to `ncvarid'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x3ae): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_nctlen':
nf_v2compat.c:(.text+0x3c5): undefined reference to `nctypelen'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x3d9): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncclos':
nf_v2compat.c:(.text+0x3f5): undefined reference to `ncclose'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x409): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncredf':
nf_v2compat.c:(.text+0x425): undefined reference to `ncredef'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x439): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncendf':
nf_v2compat.c:(.text+0x455): undefined reference to `ncendef'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x469): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncinq':
nf_v2compat.c:(.text+0x486): undefined reference to `ncinquire'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x49d): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncsnc':
nf_v2compat.c:(.text+0x4b5): undefined reference to `ncsync'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x4c9): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncabor':
nf_v2compat.c:(.text+0x4e5): undefined reference to `ncabort'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x4f9): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncdinq':
nf_v2compat.c:(.text+0x520): undefined reference to `ncdiminq'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x540): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncdren':
nf_v2compat.c:(.text+0x556): undefined reference to `ncdimrename'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x56c): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncvinq':
nf_v2compat.c:(.text+0x58f): undefined reference to `ncvarinq'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x5ac): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncvpt1':
nf_v2compat.c:(.text+0x5e5): undefined reference to `nc_inq_vartype'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x622): undefined reference to `nc_put_var1_double'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x634): undefined reference to `nc_put_var1_float'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x646): undefined reference to `nc_put_var1_int'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x658): undefined reference to `nc_put_var1_short'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x671): undefined reference to `nc_put_var1_schar'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x68c): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x693): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncvp1c':
nf_v2compat.c:(.text+0x6e2): undefined reference to `nc_inq_vartype'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x708): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x70f): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x730): undefined reference to `nc_put_var1_text'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncvpt':
nf_v2compat.c:(.text+0x774): undefined reference to `nc_inq_vartype'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x7b3): undefined reference to `nc_put_vara_double'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x7c7): undefined reference to `nc_put_vara_float'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x7db): undefined reference to `nc_put_vara_int'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x7ef): undefined reference to `nc_put_vara_short'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x80a): undefined reference to `nc_put_vara_schar'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x825): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x82c): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncvptc':
nf_v2compat.c:(.text+0x885): undefined reference to `nc_inq_vartype'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x8c7): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x8ce): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x8f0): undefined reference to `nc_inq_varndims'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x9dc): undefined reference to `nc_put_vara_text'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncvptg':
nf_v2compat.c:(.text+0xa15): undefined reference to `nc_inq_vartype'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xa2a): undefined reference to `nc_inq_varndims'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xa7a): undefined reference to `nc_put_varm_double'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xaa1): undefined reference to `nc_put_varm_float'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xac5): undefined reference to `nc_put_varm_int'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xae9): undefined reference to `nc_put_varm_short'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xb14): undefined reference to `nc_put_varm_schar'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xb33): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xb3a): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncvpgc':
nf_v2compat.c:(.text+0xb97): undefined reference to `nc_inq_vartype'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xba9): undefined reference to `nc_inq_varndims'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xbd7): undefined reference to `nc_put_varm_text'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xc12): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xc19): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncvgt1':
nf_v2compat.c:(.text+0xc65): undefined reference to `nc_inq_vartype'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xca2): undefined reference to `nc_get_var1_double'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xcb4): undefined reference to `nc_get_var1_float'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xcc6): undefined reference to `nc_get_var1_int'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xcd8): undefined reference to `nc_get_var1_short'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xcf1): undefined reference to `nc_get_var1_schar'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xd0c): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xd13): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncvg1c':
nf_v2compat.c:(.text+0xd62): undefined reference to `nc_inq_vartype'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xd7b): undefined reference to `nc_get_var1_text'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xdaf): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xdb6): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncvgt':
nf_v2compat.c:(.text+0xdf4): undefined reference to `nc_inq_vartype'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xe33): undefined reference to `nc_get_vara_double'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xe47): undefined reference to `nc_get_vara_float'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xe5b): undefined reference to `nc_get_vara_int'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xe6f): undefined reference to `nc_get_vara_short'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xe8a): undefined reference to `nc_get_vara_schar'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xea5): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xeac): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncvgtc':
nf_v2compat.c:(.text+0xf08): undefined reference to `nc_inq_vartype'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xf4d): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xf54): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xf7e): undefined reference to `nc_get_vara_text'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0xf90): undefined reference to `nc_inq_varndims'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncvgtg':
nf_v2compat.c:(.text+0x10b5): undefined reference to `nc_inq_vartype'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x10ca): undefined reference to `nc_inq_varndims'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x111a): undefined reference to `nc_get_varm_double'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x1141): undefined reference to `nc_get_varm_float'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x1165): undefined reference to `nc_get_varm_int'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x1189): undefined reference to `nc_get_varm_short'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x11b4): undefined reference to `nc_get_varm_schar'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x11d3): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x11da): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncvggc':
nf_v2compat.c:(.text+0x1237): undefined reference to `nc_inq_vartype'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x1249): undefined reference to `nc_inq_varndims'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x1277): undefined reference to `nc_get_varm_text'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x12b2): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x12b9): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncvren':
nf_v2compat.c:(.text+0x12e6): undefined reference to `ncvarrename'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x12fc): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncapt':
nf_v2compat.c:(.text+0x1337): undefined reference to `nc_put_att_double'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x1341): undefined reference to `nc_put_att_float'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x134b): undefined reference to `nc_put_att_int'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x1355): undefined reference to `nc_put_att_short'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x1367): undefined reference to `nc_put_att_schar'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x1394): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x139b): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncaptc':
nf_v2compat.c:(.text+0x13cc): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x13d3): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x13e9): undefined reference to `nc_put_att_text'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncainq':
nf_v2compat.c:(.text+0x1406): undefined reference to `ncattinq'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x141d): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncagt':
nf_v2compat.c:(.text+0x1455): undefined reference to `nc_inq_atttype'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x1492): undefined reference to `nc_get_att_double'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x14a4): undefined reference to `nc_get_att_float'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x14b6): undefined reference to `nc_get_att_int'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x14c8): undefined reference to `nc_get_att_short'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x14e1): undefined reference to `nc_get_att_schar'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x14fc): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x1503): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncagtc':
nf_v2compat.c:(.text+0x1558): undefined reference to `nc_inq_atttype'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x157f): undefined reference to `nc_advise'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x1586): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x15aa): undefined reference to `nc_inq_attlen'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x15cd): undefined reference to `nc_get_att_text'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncacpy':
nf_v2compat.c:(.text+0x1616): undefined reference to `ncattcopy'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x162d): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncanam':
nf_v2compat.c:(.text+0x1646): undefined reference to `ncattname'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x165c): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncaren':
nf_v2compat.c:(.text+0x1676): undefined reference to `ncattrename'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x168c): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncadel':
nf_v2compat.c:(.text+0x16a6): undefined reference to `ncattdel'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x16bc): undefined reference to `ncerr'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: /glade/scratch/heinzell/hpc-stack/source/install/intel-19.1.1/mpt-2.22/netcdf/4.7.4/lib/libnetcdff.a(nf_v2compat.o): in function `c_ncsfil':
nf_v2compat.c:(.text+0x16d6): undefined reference to `ncsetfill'
/usr/lib64/gcc/x86_64-suse-linux/4.8/../../../../x86_64-suse-linux/bin/ld: nf_v2compat.c:(.text+0x16ec): undefined reference to `ncerr'
makefile:506: recipe for target 'madis_ingest' failed
make: *** [madis_ingest] Error 1
BUILD FAIL!  Lib: madis Error:2
```

--- a/modulefiles/compiler/compilerName/compilerVersion/bufr/bufr.lua
+++ b/modulefiles/compiler/compilerName/compilerVersion/bufr/bufr.lua
@@ -31,7 +31,10 @@ setenv("BUFR_LIB8_DA", pathJoin(base,"${CMAKE_INSTALL_LIBDIR}","libbufr_8_DA.a")
 setenv("BUFR_LIBd_DA", pathJoin(base,"${CMAKE_INSTALL_LIBDIR}","libbufr_d_DA.a"))
 
 prepend_path("PATH", pathJoin(base,"bin"))
-prepend_path("PYTHONPATH", pathJoin(base,"${CMAKE_INSTALL_LIBDIR}","python${PYTHON_VERSION}/site-packages"))
+local pydir = pathJoin(base,"${CMAKE_INSTALL_LIBDIR}","python${PYTHON_VERSION}/site-packages")
+if (isDir(pydir)) then
+  prepend_path("PYTHONPATH", pydir)
+end
 
 whatis("Name: ".. pkgName)
 whatis("Version: " .. pkgVersion)

--- a/stack_helpers.sh
+++ b/stack_helpers.sh
@@ -5,7 +5,7 @@ function update_modules {
   local modpath=$1
   local name=$2
   local version=$3
-  local python_version=$4
+  local py_version=${4:-}
   case $modpath in
     core )
       local tmpl_file=$HPC_STACK_ROOT/modulefiles/core/$name/$name.lua
@@ -49,7 +49,7 @@ EOF
     CMAKE_INSTALL_LIBDIR=$(cat $testdir/cmake_install_libdir.txt)
     CMAKE_OPTS="-DCMAKE_INSTALL_LIBDIR=${CMAKE_INSTALL_LIBDIR}"
     CMAKE_OPTS+=" -DTMPL_FILE=$tmpl_file -DVERSION=$version"
-    [[ -n "$4" ]] && CMAKE_OPTS+=" -DPYTHON_VERSION=$python_version"
+    [[ -n "${py_version:-}" ]] && CMAKE_OPTS+=" -DPYTHON_VERSION=$py_version"
     # Install the module with configure_file, replacing ${CMAKE_INSTALL_LIBDIR} (and potentially other variables)
     # with the actual value for that system
     $SUDO cmake $CMAKE_OPTS -P ${HPC_STACK_ROOT}/cmake/configure_module.cmake

